deployment/README.md

```diff
@@ -8,149 +8,349 @@ proxy](https://doc.traefik.io/traefik/) as a reverse proxy, which listens
 on port 80 and 443.  Based on some labels, traefik routes the domains to the
 corresponding docker containers.
 
+## Requirements
+
+This guide requires a Linux-system, where `docker` and `docker-compose` are installed.
+Ensure, that your system is up to date.
+
+> TODO
+
+```bash
+apt install docker.io docker-compose pwgen
+```
+
 ## Before Getting Started
 
-The guide and example configuration assumes one domain, which points to the
-server's IP address. This documentation uses `portal.example.com` as an
-example. The API is hosted at `https://portal.example.com/api`, while the main
-frontend is reachable at the domain root.
+The example configurations assume two domains, which points to the
+server's IP address. This documentation uses `portal.example.com` and
+`login.example.com`. The API is hosted at `https://portal.example.com/api`, 
+while the main frontend is reachable at the domain root.
```

Comment:
This isn't really accurate anymore, as the API exposes the frontend itself. It's only reusing that URL, but it is not really two services.